src/main/scala/cromwell/binding/Task.scala

```diff
@@ -52,27 +31,121 @@ object Task {
  * Represents a `task` declaration in a WDL file
  *
  * @param name Name of the task
- * @param command Abstract command defined in the `command` section
+ * @param declarations Any declarations (e.g. String something = "hello") defined in the task
+ * @param commandTemplate Sequence of command pieces, essentially a parsed command template
  * @param outputs Set of defined outputs in the `output` section of the task
+ * @param ast The syntax tree from which this was built.
+ * @param backendType which backend this task is meant to run on.
  */
 case class Task(name: String,
                 declarations: Seq[Declaration],
-                command: Command,
+                commandTemplate: Seq[CommandPart],
                 outputs: Seq[TaskOutput],
                 ast: Ast,
                 backendType: BackendType) extends Executable {
+  val ws = Pattern.compile("[\\ \\t]+")
```

Comment:
If this is a static constant it should be in the companion object and named `Ws`